service/settings/base.py

```diff
@@ -155,6 +155,10 @@
     'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',),
 }
 
+# REST FRAMEWORK
+JSON_API_PLURALIZE_TYPES = True
+JSON_API_FORMAT_TYPES = 'camelize'
+JSON_API_FORMAT_KEYS = 'dasherize'
```

Comment:
Make ember not care about having to mess around with nameing conventions- just do it in the backend :)